src/cgame/cg_local.h

```diff
@@ -1415,16 +1415,14 @@ struct rocketDataSource_t
 
 	int selectedTeamIndex;
 
-	int selectedHumanSpawnItem;
-	int selectedAlienSpawnClass;
+	int selectedSpawnOptions[ NUM_TEAMS ];
 
 	int armouryBuyList[3][ ( WP_LUCIFER_CANNON - WP_BLASTER ) + UP_NUM_UPGRADES + 1 ];
 	int selectedArmouryBuyItem[3];
 	int armouryBuyListCount[3];
 
 	BoundedVector<int, PCL_NUM_CLASSES> alienEvolveList;
-	BoundedVector<int, BA_NUM_BUILDABLES> humanBuildList;
```

Comment:
alien/humanBuildList are dead code, just delete them

Reply:
This is what I intended to do originally, but then I thought that maybe they should be repaired and used, this would fix few problems such as having an apparent choice to birth with things that are disabled or not unlocked yet (if someday someone wants to allow birth options that change with momentum).

Reply:
We already have that information available in a data source and used by the circlemenus. The stuff here is a remnant of the deleted code for the old circlemenus.